code/modules/mob/living/simple_animal/abnormality/teth/punishing_bird.dm

```diff
@@ -193,6 +193,14 @@
 	..()
 	Retaliate(user)
 
+/mob/living/simple_animal/hostile/abnormality/punishing_bird/breach_effect(mob/living/carbon/human/user)
+	..()
+	addtimer(CALLBACK(src, .proc/kill_bird()), 240 SECONDS)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/punishing_bird/proc/kill_bird()
+	Destroy()
```

Comment:
It is certainly NOT a good way to handle it.

Reply:
Yeah, it's entirely a temp fix to the issue. I'd love if he could path back and sit back down but I don't even know if we have a codebase FOR abnormalities returning calmly on their own.